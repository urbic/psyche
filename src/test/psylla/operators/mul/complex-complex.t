1 2 complex 3 4 complex mul ?
