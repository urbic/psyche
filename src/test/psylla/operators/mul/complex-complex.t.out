%complex=-5.0;10.0%
